src/printer.js

```diff
@@ -1209,13 +1209,20 @@ function genericPrintNoParens(path, options, print) {
       else parts.push("default:");
 
       if (n.consequent.find(node => node.type !== "EmptyStatement")) {
-        const cons = path.call(
-          function(consequentPath) {
-            return printStatementSequence(consequentPath, options, print);
-          },
-          "consequent"
-        );
-
+        const cons = path.call(consequentPath => {
+          let printed = [];
+          path.map(p => {
+            const value = p.getValue();
+            const parent = p.getParentNode();
```

Comment:
can you avoid creating those two variables, you are only using them once and it makes it more verbose than needs be